crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -883,12 +883,10 @@ fn manage_dlc_manager<S: TenTenOneStorage + 'static, N: Storage + Sync + Send +
             loop {
                 tracing::trace!("Started periodic dlc manager check");
```

Comment:
Since you've changed this it would be nice to align the whole function including the logs to reflect that this task is now solely about calling `periodic_chain_monitor` i.e. feeding new blocks to the `rust-dlc` `ChainMonitor`.